### PR TITLE
Frontend tests/Type Providers: remove 'withOutOfProcessTypeProviders'

### PR DESCRIPTION
**Pull request by DedSec256**
Since we host type providers out-of-process by default

### CODE REVIEW

**Comment by DedSec256** [comment]
We don't need the number of type provider processes assertion, since we can detect the problem from the logs

File: rider-fsharp/src/test/kotlin/typeProviders/TypeProvidersRuntimeTest.kt [hunk 1]
```diff
@@ -22,9 +21,9 @@ class TypeProvidersRuntimeTest : BaseTestWithSolution() {
 
     @Test
     @TestEnvironment(
-        toolset = ToolsetVersion.TOOLSET_16,
-        coreVersion = CoreVersion.DOT_NET_CORE_3_1,
-        solution = "TypeProviderLibrary")
+            toolset = ToolsetVersion.TOOLSET_16,
```
Comment (auduchinok) [inline rider-fsharp/src/test/kotlin/typeProviders/TypeProvidersRuntimeTest.kt:24]: Could you please move formatting changes to a separate PR?

**Review by auduchinok — APPROVED** [review]
